reactor-kafka-tools/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2016-2018 Pivotal Software Inc, All Rights Reserved.
```

Comment:
It's kinda weird to have a Copyright in the non-Java files, but OK if it is what we have so far in this project.
Anyway: I see you have modified Copyright to VMware in other place. So, this one might be affected, too.

On the other hand the Log4J 1.x is out of life for a while already: https://logging.apache.org/log4j/1.2/ 😄 

Different issue though...